main.tf

```diff
@@ -153,15 +161,15 @@ resource "aws_iam_role" "ecs_task" {
   count = local.create_task_role ? 1 : 0
 
   name                 = module.task_label.id
-  assume_role_policy   = join("", data.aws_iam_policy_document.ecs_task.*.json)
+  assume_role_policy   = join("", data.aws_iam_policy_document.ecs_task[*].json)
   permissions_boundary = var.permissions_boundary == "" ? null : var.permissions_boundary
   tags                 = var.role_tags_enabled ? module.task_label.tags : null
 }
 
 resource "aws_iam_role_policy_attachment" "ecs_task" {
   for_each   = local.create_task_role ? toset(var.task_policy_arns) : toset([])
   policy_arn = each.value
-  role       = join("", aws_iam_role.ecs_task.*.id)
+  role       = join("", aws_iam_role.ecs_task[*].id)
```

Comment:
```suggestion
  role       = aws_iam_role.ecs_task[0].id
```

Reply:
as long as we won't run into issues I would prefer to keep it like in the source

Reply:
As we've discussed in the meeting, i'm fine with keeping it as-is to stay compatible and have it easy to pull in upstream changes from cloudposse